### DISCUSSION TITLE
Is Dependency Track v4.4 set to be released soon?

### DISCUSSION THREAD

**Discussion by lukas-adoma** [DependencyTrack/dependency-track#1401] [General]
I noticed the v4.4 milestone issues have been set to close. Does this mean there will be a Dependency Track v4.4 release coming soon? I'm very excited about the fixed issues and look forward to this new version.

Thanks.

**Comment by stevespringett** [comment]
Frontend was just released. Working on API Server.

**Reply by lukas-adoma** [reply]
Fantastic! Glad to hear.
